.github/workflows/docs-pull.yaml

```diff
@@ -0,0 +1,19 @@
+name: Pull remote documents
+
+on:
+  workflow_dispatch: # Triggered manually through the "Start workflow" button
+jobs:
+  list-documents:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # Check out the repository
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: npm install octokit minimatch
```

Comment:
Please also use yarn to install the packages here, otherwise there could be some unwanted behaviour with it in the next step